test/test_dataset.py

```diff
@@ -3,14 +3,15 @@
 
 from torch.testing._internal.common_utils import (TestCase, run_tests)
 
-from torch.utils.data.datasets import (ListDirFilesIterableDataset)
+from torch.utils.data.datasets import (ListDirFilesIterableDataset, LoadFilesFromDiskIterableDataset)
 
 def create_temp_dir_and_files():
-    temp_dir = tempfile.TemporaryDirectory()
+    # Note: the temp dir and files within it will be deleted in tearDown()
+    temp_dir = tempfile.TemporaryDirectory()  # noqa: P201
     temp_dir_path = temp_dir.name
-    temp_file1 = tempfile.NamedTemporaryFile(dir=temp_dir_path, delete=False)
-    temp_file2 = tempfile.NamedTemporaryFile(dir=temp_dir_path, delete=False)
-    temp_file3 = tempfile.NamedTemporaryFile(dir=temp_dir_path, delete=False)
+    temp_file1 = tempfile.NamedTemporaryFile(dir=temp_dir_path, delete=False)  # noqa: P201
+    temp_file2 = tempfile.NamedTemporaryFile(dir=temp_dir_path, delete=False)  # noqa: P201
+    temp_file3 = tempfile.NamedTemporaryFile(dir=temp_dir_path, delete=False)  # noqa: P201
```

Comment:
Can you please commend this noqa

Reply:
These #noqa: P201 is removed in the later stack PRs, i changed another way to create these files.